libcloud/backup/types.py

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__all__ = [
+    'Provider',
+    'BackupTargetType',
+    'BackupTargetJobStatusType'
+]
+
+
+class Provider(object):
+    DUMMY = 'dummy'
+    EBS = 'ebs'
+    GCE = 'gce'
+    DIMENSIONDATA = 'dimensiondata'
+
+
+class BackupTargetType(object):
+    """
+    Backup Target type.
+    """
+
+    VIRTUAL = 'Virtual'
```

Comment:
Minor thing I just noticed - for consistency with other enums, please make sure the enum values are all lowercase (`VIRTUAL = 'virtual`, etc.).